cs.json

```diff
@@ -0,0 +1,348 @@
+            }
+        },
+        "upcoming" : {
+            "title" : "Nadcházející kosmetika",
+            "line1" : "Všechny nevydané položky ve Fortnite: Battle Royale."
+        },
+        "history" : {
+            "list" : {
+                "title" : "Historie obchodu",
+                "line1" : "Zde najdete předchozí rotace obchodu pro Fortnite: Battle Royale.",
+                "line2" : "Klikněte na jednu pro zobrazení celé rotace nebo rolujte myší pro zobrazení dalších položek dle počtu dnů."
+            },
+            "view" : {
+                "title" : "Rotace obchodu",
+                "line1" : "Toto je rotace obchodu ze dne [date] pro Fortnite: Battle Royale. ",
+                "line2" : "Kliknutím na kosmetiku zobrazíte další informace o vybrané položce.",
+                "today-button" : "Dnešní rotace"
+            }
+        },
+        "news" : {
+            "title" : "Novinky",
+            "line1" : "Aktuální informační kanál o hře pro Fortnite: Battle Royale from Epic Games.",
+            "error" : "Nepodařilo se nám získat novinky od Fortnite, zkusíme to znovu za chvíli.",
+            "more-ingame-button" : "Načti více"
+        },
+        "randomiser" : {
+            "title" : "Randomizér",
+            "line1" : "Zvolí náhodný oděv ze všech dostupných položek.",
+            "chooseText" : "Klikněte pro výběr",
+            "randomise-all" : "Randomizovat vše"
+        },
+        "item" : {
+            "unreleased" : "Nevydané",
+            "description" : "Popis",
+            "history" : {
+                "title" : "Historie obchodu",
+                "release-never" : "Datum vydání: Nikdy",
+                "release-date" : "Datum vydání:",
+                "last-never" : "Naposledy viděno: Nikdy",
+                "last-date" : "Naposledy viděno:",
+                "occurrences" : "Počet výskytů:",
+                "introduced" : "Představeno v:"
+            },
+            "reminder-count" : "Uživatelé s [upozorněními](/reminders) na Discordu:",
+            "app-reminder-count" : "Uživatelé s upozorněními v [naší aplikaci](/app):",
+            "add-reminder" : "Přidat upozornění",
+            "remove-reminder" : "Smazat upozornění",
+            "has-reminders" : "Máte nastavené upozornění pro tuto položku.",
+            "no-reminders" : "Upozornění nejsou dostupné pro tuto položku.",
+            "images" : "Obrázky",
+            "images-icon" : "Ikony",
+            "images-png" : "PNG",
+            "images-gallery" : "Galerie",
+            "images-featured" : "Doporučené",
+            "images-video" : "Pohled ve hře",
+            "occurrences-table" : {
+                "title" : "Počet výskytů",
+                "th-date" : "Datum",
+                "th-days" : "Před [date] dny", //This one is tricky, because it should be (Před 1 dnem, Před 2 dny, Před 3 dny, ...) as in (1 day ago, 2 days ago, 3 days ago, ...), but it could also be (x Dnů zpět) which is not incorrect either
```

Comment:
We can't use a `[date]` placeholder here because this text is appended to the number. I.e. the output would be "3 Před [date] dny" when it was 3 days ago. So I guess we should go with "Dnů zpět" here.